src/Tizen.Security.TEEC/Tizen.Security.TEEC/Libteec.cs

```diff
@@ -328,6 +328,7 @@ public sealed class Value : BaseParameter<TEFValueType>
     /// <since_tizen> 3 </since_tizen>
     public sealed class Session
     {
+        private bool closed = false;
```

Comment:
Close only valid after call of Open32/Open64, 

So, `closed` value initially should be true, 
and It changed to false in Open32/Open64

Reply:
Done